RemoteControl/RemoteControl.cpp

```diff
@@ -35,6 +35,10 @@
 
 #define RF4CE_IEEE_MAC_ADDRESS_STR_MAX    24
 
+#define IARM_FACTORY_RESET_TIMEOUT  (15 * 1000)  // 15 seconds, in milliseconds
+#define IARM_IRDB_CALLS_TIMEOUT     (10 * 1000)  // 10 seconds, in milliseconds
+
+
```

Comment:
Pls update minor patch version to 3 to match the CHANGELOG. This would then show up in Controller.status API for RemoteControl. 

#define API_VERSION_NUMBER_MINOR 0